docs/source/setup/Developer_Setup.md

```diff
@@ -52,4 +52,4 @@ pip install chronon-ai
 
 The main directory and the python library is all that is required to run the developer steps mentioned at the top of this document.
 
-Note that depending on your preferred developer experience, users may be authoring Chronon objects locally then syncing them to a remote machine for execution on the spark cluster. There a number of ways to enable this, the simplest being a `sync` command that copies over files from local to remote. However, there could be better flows from a UX perspective, depending on your existing infrastructure for running jobs on the cluster. Feel free to reach out in the [community Discord channel](https://discord.gg/GbmGATNqqP) if you wish to discuss ways to implement a better developer experience.
+Note that depending on your preferred developer experience, users may be authoring Chronon objects locally then syncing them to a remote machine for execution on the spark cluster. There a number of ways to enable this, the simplest being a `sync` command that copies over files from local to remote. However, there could be better flows from a UX perspective, depending on your existing infrastructure for running jobs on the cluster. Feel free to reach out in the [community Slack workspace](https://join.slack.com/t/chrononworkspace/shared_invite/zt-2r621b6hw-pm552u71Y257Vtpt4RTiyg) if you wish to discuss ways to implement a better developer experience.
```

Comment:
@piyush-zlai sometimes the link will expire. Do you know if this is a long term link?

Reply:
@pengyu-hou - I actually picked up a link that has no expiry. We might hit a cap though of the number of folks in the workspace but hopefully we'll see that coming before.

Reply:
Sounds good. Please rebase it after my recent CI fix.